i/influxdb/build_info.json

```diff
@@ -1,14 +1,15 @@
 {
-  "maintainer": "IBM-Currency-Helper",
+  "maintainer": "vinodk99",
   "package_name": "influxdb",
-  "github_url": "https://github.com/influxdata/influxdb.git",
-  "version": "v2.5.1",
-  "default_branch": "master",
-  "build_script": "influxdb_ubi_8.5.sh",
-  "package_dir": "i/influxdb/Dockerfiles/",
-  "docker_cmd": "docker build -t ${package_name}:$PACKAGE_VERSION ${dir}",
-  "v2.*.*": {
-    "dir": "v2.5.1_ubi_8.5",
-    "build_script": "influxdb_ubi_8.5.sh"
+  "github_url": "https://github.com/influxdata/influxdb",
+  "version": "v2.7.6",
+  "default_branch": "main",
+  "package_dir": "i/influxdb",
+  "build_script": "influxdb_ubi_9.3.sh",
+  "validate_build_script": true,
+  "use_non_root_user": "false",
+  "docker_build": false,
```

Comment:
@vinodk99  is docker build not working for this package with ubi 9?

Reply:
@nikhil-kalbande added ubi9 dockerfile for influxdb as suggested